### ISSUE TITLE
The main navigation bar is not navigable using a keyboard

### ISSUE THREAD

**Issue by ADubhlaoich** [EuroPython/website#694]
## Description
**As an** attendee, **I want** the navigation to be logically traversable, **so that** I only need a keyboard for navigation.

## Acceptance criteria
A user should be able to navigate through the entirety of the navigation bar with a keyboard alone.

Within the context of the navigation bar, this means that the tab order should go left to right, then top to bottom.

## Additional information
[WCAG 1.3.2 Meaningful Sequence](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence)
[WCAG 2.1 Keyboard Accessible](https://www.w3.org/WAI/WCAG21/Understanding/keyboard-accessible)
[WCAG 2.4.3 Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)

**Comment by ADubhlaoich** [comment]
I phrased this ticket in a keyboard-centric way since it's what I tested it with, but the tab order is also used by [adaptive switches](https://ialabs.ie/adaptive-switches-and-how-they-help-people-with-disabilities/).

This is the current tab order:

- Skip to main link
- Homepage
- Code of conduct
- Tickets

The next three links are Volunteers, Tickets and Sponsors as part of the actual page content "under the fold". It looks as though the entire navigation menu is completely avoided.
